pkg/apb/client.go

```diff
@@ -11,9 +11,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	restclient "k8s.io/client-go/rest"
 
+	"github.com/openshift/ansible-service-broker/pkg/clients"
```

Comment:
Wow I never realized how ugly our import is :( not really a problem with this PR, just something I noticed finally. Would be cool if we could get: `openshift.io/asb/`